packages/containers/src/ActionButton/ActionButton.connect.js

```diff
@@ -28,14 +28,6 @@ export function mergeProps(stateProps, dispatchProps, ownProps) {
 export function ContainerActionButton(props) {
 	const newProps = Object.assign({}, props);
 
-	if (typeof props.overlayComponent === 'string' && props.overlayComponent) {
```

Comment:
Now, the inject is in the component OverlayTrigger

Reply:
I don't see the code for that